src/Service/SF1500/SF1500XMLBuilder.php

```diff
@@ -0,0 +1,415 @@
+<?php
+
+/**
+ * This file is part of itk-dev/serviceplatformen.
+ *
+ * (c) 2020 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
+namespace ItkDev\Serviceplatformen\Service\SF1500;
+
+/**
+ * Helper class for SF1500 Organisation.
+ */
+class SF1500XMLBuilder
+{
+    /**
+     * Builds XML body for bruger laes.
+     */
+    public function buildBodyBrugerLaesXML($uuid): string
```

Comment:
Consider passing all XML parameter values in this class though https://www.php.net/manual/en/function.htmlspecialchars.php for a little added safety, i.e.

```php
$uuid = $this->xmlEncode($uuid);
…
$uuids = array_map([$this, 'xmlEncode'], $uuids);
…
private function xmlEncode(string $value) {
  return htmlspecialchars($value, ENT_QUOTES| ENT_XML1);
}
…
}
```

Reply:
Added extra safety via `htmlspecialchars`.